examples/kubernetes/addons/prometheus/01-cilium.yaml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
Any reason to have to many different files?

Reply:
Yes, one was a bit big, and each file has his own role. Kube-metrics, namespace, prometheus, grafana and grafana-dashboards.

Reply:
Yeah, but why can't we have all those multiple files **and** one big file which will be easier to wget and use? We do exactly the same with Cilium yaml manifests by creating templates for small files and then generating the big file. I think I'd be cool to have separate templates for prometheus and to extend `Makefile` to generate prometheus manifests.

Reply:
@eloycoto yes but you are requiring the user to download all files locally in order to run `kubectl apply -f`, if they were in a single file it will make is easier for users to deploy it using `kubectl create -f https://raw.github...`

Also, it should be better to say run `kubectl create` instead of `kubectl apply` as, by doing apply, you might accidentally overwrite something they are running in their cluster.